source/blog/2018-11-09-the-ember-times-issue-72.md

```diff
@@ -26,8 +26,13 @@ _”Pick the topic you feel most passionate about. This is your conference.”_
 
 ---
 
-## [SECTION TITLE](#section-url)
+## [Yo Dawg, I Heard You Like RFCs](https://github.com/emberjs/rfcs/pull/300)
+There has been an update to the RFC about the RFC process itself. The RFCs has been crucial in defining **the future of Ember** and that is still the case.
+However, in the process of reading and merging all of these RFCs the Core Team has discovered some shortcomings of the current RFC process as well as new requirements.
```

Comment:
Suggestion only (feel free to ignore), caveat being I'm pretty heavy handed with commas myself 

```suggestion
However, in the process of reading and merging all of these RFCs, the Core Team has discovered some shortcomings of the current RFC process as well as new requirements.
```

Reply:
@amyrlam Thanks, this is really helpful! I only know Danish comma rules and they don't translate well.